bin/build_sdb.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+'''
+Builds Sample Databases (.sdb files)
```

Comment:
Might be worth explaining a bit more the thing here, since I suppose this is going to be the more user-facing component.

Reply:
Line 43 is not helping?

Reply:
Well, it's not saying much much more :/

Reply:
Added a bit more wording.